Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/ControllerHelper.cs

```diff
@@ -15,8 +17,9 @@ namespace EdFi.Ods.Tests.EdFi.Ods.Api.Controllers
 {
     public static class ControllerHelper
     {
-        public static TokenController CreateTokenController(ClientCredentialsTokenRequestProvider tokenRequestProvider)
+        public static TokenController CreateTokenController(IClientAppRepo clientAppRepo,IApiClientAuthenticator apiClientAuthenticator)
```

Comment:
probably want to add the headers as a parameter here too

Reply:
Headers are passed to CreateControllerContext() which is getting called in ActAsync()